src/style.css

```diff
@@ -0,0 +1,49 @@
+body {
```

Comment:
- [ ] Please match the [wireframe ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)positioning provided by Microverse to comply with the project requirements.